lib/onstomp/client.rb

```diff
@@ -9,7 +9,7 @@ class OnStomp::Client
   include OnStomp::Interfaces::ReceiptManager
   include OnStomp::Interfaces::SubscriptionManager
   include OnStomp::Components::Scopes
-  
+
```

Comment:
Bah, sorry - I guess my editor stripped out the trailing whitespace all over this file.

Reply:
No problem, it could use the cleanup!